charts/aws-vpc-cni/README.md

```diff
@@ -48,15 +48,15 @@ The following table lists the configurable parameters for this chart and their d
 | `minimumWindowsIPTarget`| Minimum IP target value for Windows prefix delegation   | `3`                                 |
 | `branchENICooldown`     | Number of seconds that branch ENIs remain in cooldown   | `60`                                |
 | `fullnameOverride`      | Override the fullname of the chart                      | `aws-node`                          |
-| `image.tag`             | Image tag                                               | `v1.16.4`                           |
+| `image.tag`             | Image tag                                               | `v1.17.1`                           |
```

Comment:
How about the new env for strict mode?

Reply:
We don't define all env vars in this readme. I'll add the env variable in the main readme